webapp/src/screens/About.tsx

```diff
@@ -114,134 +132,132 @@ export default function About() {
+							>
+								Our team’s motivation for creating QBot was to be a student’s fastest way of getting
+								accurate answers to the most commonly asked questions about the class they are taking with
+								their Professor.
+							</Typography>
+						</Box>
+					</Container>
+				</Box>
 
-        <Container sx={{ py: 3, marginLeft: 0 }} maxWidth="md">
-          {/* End hero unit */}
-          <Grid container spacing={4}>
-            {cards.map((card) => (
-              <Grid item key={card.id} xs={12} sm={6} md={4}>
-                <Card
-                  sx={{ height: '100%', display: 'flex', flexDirection: 'column' }}
-                >
-                  <CardMedia
-                    component="div"
-                    sx={{
+				<Box
+					sx={{
+						bgcolor: 'background.paper',
+						pt: 0,
+						pb: 0,
+					}}
+				>
+					<Container maxWidth="md" sx={{
+						marginLeft: 0
+					}}>
+						<Typography
+							variant="h3"
+							align="left"
+							color="text.primary"
+							gutterBottom
+						>
+							Published Paper
+						</Typography>
+						<Divider />
+						<Box
+							sx={{
+								pt: 0,
+								pb: 2
+							}}
+						>
+							<Typography
+								variant="body1"
+								align="left"
+								color="text.secondary"
+								maxWidth="md"
+							>
+								The link to the paper we wrote about our project will be posted here once published.
+							</Typography>
+						</Box>
+						<Typography
+							variant="h3"
+							align="left"
+							color="text.primary"
+							gutterBottom
+						>
+							Project Team
+						</Typography>
+						<Divider />
+					</Container>
+				</Box>
 
+				<Container sx={{ py: 3, marginLeft: 0 }} maxWidth="md">
+					{/* End hero unit */}
+					<Grid container spacing={4}>
+						{cards.map((card) => (
+							<Grid item key={card.id} xs={12} sm={6} md={4}>
+								<Card
+									sx={{ height: '100%', display: 'flex', flexDirection: 'column' }}
+								>
+									<CardMedia
+										component="div"
+										sx={{
 											height: "200px"
-                    }}
-                    // TODO: make images work.
-                    image={card.image}
-                  />
-                  <CardContent sx={{ flexGrow: 1 }}>
-                    <Typography gutterBottom variant="h5" component="h2">
-                      {card.name}
-                    </Typography>
-                    <Typography sx={{ color: "text.secondary" }}>
-                      {card.description}
-                    </Typography>
-                  </CardContent>
-                  <CardActions>
-                    <IconButton aria-label="LinkedIn" href={card.linkedIn} >
+										}}
+										image={card.image}
+									/>
+									<CardContent sx={{ flexGrow: 1 }}>
+										<Typography gutterBottom variant="h5" component="h2">
+											{card.name}
+										</Typography>
+										<Typography sx={{ color: "text.secondary" }} whiteSpace="pre">
```

Comment:
This is the only functional change in the html, all the other changes are linting.
The `whiteSpace` parameter enforces line breaks in the description text